src/stylesheets/components/_banner.scss

```diff
@@ -30,3 +30,8 @@
 .usa-banner__content .usa-banner__guidance:last-child .usa-banner__icon {
   background-image: url('#{$theme-image-path}/icon-https.svg');
 }
+
+.work-in-progress {
+  margin-right: 5px;
+  text-align: right;
```

Comment:
Instead of creating a custom class, would prefer to use existing USWDS classes applied directly to the html element.

USWDS:

.text-right{
  text-align:right;
}

For margin, can we use .margin-right-05 (actual: 4px)?